core/modules/layout/css/layout.admin.css

```diff
@@ -5,6 +5,13 @@
 .layout-list .layout-group td:first-child {
   font-weight: bold;
 }
+.layout-list .layout-group td.missing-path,
+.layout-list .layout-group td.missing-path a {
+  color: red;
```

Comment:
To my knowledge it's required to use the format `#ff0000` to conform to the coding standards. But using color is a bit tricky here, anyway because of contrast.

Reply:
Thanks, I'll make that change. Maybe try a slightly brighter red to improve the contrast.